RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHGC_cff.py

```diff
@@ -1,3 +1,4 @@
 from RecoParticleFlow.PFClusterProducer.particleFlowRecHitHGC_cfi import *
 
-particleFlowRecHitHGCSeq = cms.Sequence( particleFlowRecHitHGC )
+particleFlowRecHitHGCTask = cms.Task( particleFlowRecHitHGC )
+particleFlowRecHitHGCSeq = cms.Sequence( particleFlowRecHitHGCTask )
```

Comment:
is the sequence still needed? From a very quick look into CMSSW it is only referenced in `RecoLocalCalo/Configuration/python/hgcalLocalReco_cff.py` that you are moving to Task.

Reply:
Thank you. I will remove `particleFlowRecHitHGCSeq = cms.Sequence( particleFlowRecHitHGCTask ) `